test/k8sT/fqdn.go

```diff
@@ -112,7 +112,7 @@ var _ = Describe("K8sFQDNTest", func() {
 		_ = kubectl.Exec(fmt.Sprintf("%s delete --all cnp", helpers.KubectlCmd))
 	})
 
-	SkipItIf(helpers.SkipQuarantined, "Restart Cilium validate that FQDN is still working", func() {
+	It("Restart Cilium validate that FQDN is still working", func() {
```

Comment:
Note: for testing only, revert before marking ready for review. The test should probably remain in quarantine for a while after this so verify the fix is working.